execution.bs

```diff
@@ -4456,28 +4501,27 @@ enum class forward_progress_guarantee {
 
     template &lt;class Sigs, class E>
       concept <i>valid-completion-signatures</i> = // exposition only
-        <i>is-instance-of</i>&lt;Sigs, completion_signatures> ||
-        (
-          same_as&lt;Sigs, dependent_completion_signatures&lt;no_env>> &&
-          same_as&lt;E, no_env>
-        );
+        <i>is-instance-of</i>&lt;Sigs, completion_signatures>;
+
+    template &lt;class S>
+      concept sender =
+        requires (const remove_cvref_t&lt;S>& s) {
+          { get_attrs(s) } -> queryable;
+        } &&
+        move_constructible&lt;remove_cvref_t&lt;S>> &&
+        constructible_from&lt;remove_cvref_t&lt;S>, S>;
 
     template &lt;class S, class E>
-      concept <i>sender-base</i> = // exposition only
+      concept sender_in =
```

Comment:
The `concept sender_in` in the synopsis has `class E = empty-env`, but this one does not.